docs/docsite/rst/scenario_guides/guide_aci.rst

```diff
@@ -1,7 +1,7 @@
 .. _aci_guide:
 
-Cisco ACI Guide
-===============
+Cisco ACI Ansible Guide
```

Comment:
I am not in favor of this.

It is stored on docs.ansible.com, it has Ansible in large at the top, the page title already is "Cisco ACI Guide — Ansible Documentation", so there is absolutely no need to add it again in (every guide's) title IMO.

Reply:
This was a recommendation from our SEO expert to improve indexing and make the guides easy to find.

Reply:
Scenario Guides, User Guides, Technology Guides doesn't set people on the wrong foot but I feel Integration Guides do. I would think it's about developing plugins/inventories/modules to integrate some technology into Ansible. And not about *managing* infrastructure.

I am not so worried for people looking on Google for 2 keywords (e.g. Ansible and VMware) as they will no doubt end up on the right pages thanks to Google PageRank (i.e. they will not be distracted), but I am worried that the Documentation index becomes a high-level list of meaningless words where it's unclear what to click to find something.

IMO "Integration" is not what someone would be looking for (so it doesn't add anything to SEO, if anything it makes the other keywords worth less in the general scheme of things).

PS Looking at the list, probably both VMware and Networking belong to these "Technology guides". Networking calls it the "Network Automation Guide" so why not call everything "Automation Guides" ? ACI Automation Guide, VMware Automation Guide, etc...

Reply:
Searching for Ansible and ACI makes it the first hit. Ansible is already in the title once and in the domain. You cannot make it better than that.